src/java.base/share/classes/jdk/classfile/CodeBuilder.java

```diff
@@ -330,6 +330,22 @@ sealed interface CatchBuilder permits CatchBuilderImpl {
          */
         CatchBuilder catching(ClassDesc exceptionType, Consumer<BlockCodeBuilder> catchHandler);
 
+        /**
+         * Adds a catch block that catches an exception of the given type.
```

Comment:
```suggestion
         * Adds a catch block that catches one or more exceptions of the given types.
```
Likewise sentences need updating

Reply:
fixed, thanks!